http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/KubernetesAppConfig.scala

```diff
@@ -14,7 +15,7 @@ import org.broadinstitute.dsde.workbench.leonardo.{
 }
 import org.broadinstitute.dsp.{ChartName, ChartVersion}
 
-sealed trait GkeAppConfig {
+sealed trait KubernetesAppConfig {
```

Comment:
I wanted to reuse this trait for configs -- renamed it to make it a little more general.